### DISCUSSION TITLE
First day of the week

### DISCUSSION THREAD

**Discussion by anousss007** [ng-matero/extensions#346] [Q&A]
Hi,

Little suggestion : it would be nice to have an input to change the first day of the week. 

By default, it's Sunday, in Europe we mostly use Monday as first day. So I guess a boolean should be fine as well in this scenario.

Thank you for the high quality of this lib !

**Comment by nzbin** [comment]
```ts
  constructor(private dateAdapter: DateAdapter<Date>){}

  ngOnInit() {
    this.dateAdapter.getFirstDayOfWeek = () => 1;
  }
```
